examples/t4rec_paper_experiments/README.md

```diff
@@ -65,7 +65,7 @@ Below is the updated command to reproduce the experiment [TRANSFORMERS WITH MULT
 ```bash
 DATA_PATH=~/transformers4rec_paper_preproc_datasets/ecom_rees46/
 FEATURE_SCHEMA_PATH=datasets_configs/ecom_rees46/rees46_schema.pbtxt
-CUDA_VISIBLE_DEVICES=0 python3 -m t4r_paper_repro.transf_exp_main --output_dir ./tmp/ --overwrite_output_dir --do_train --do_eval --validate_every 10 --logging_steps 20 --save_steps 0 --data_path $DATA_PATH --features_schema_path $FEATURE_SCHEMA_PATH --fp16 --data_loader_engine nvtabular --start_time_window_index 1 --final_time_window_index 30 --time_window_folder_pad_digits 4 --model_type xlnet --loss_type cross_entropy --per_device_eval_batch_size 512 --similarity_type concat_mlp --tf_out_activation tanh --inp_merge mlp --learning_rate_warmup_steps 0 --learning_rate_schedule linear_with_warmup --hidden_act gelu --num_train_epochs 10 --dataloader_drop_last --compute_metrics_each_n_steps 1 --session_seq_length_max 20 --eval_on_last_item_seq_only --mf_constrained_embeddings --layer_norm_featurewise --attn_type bi --mlm --input_features_aggregation concat --per_device_train_batch_size 256 --learning_rate 0.00020171456712823088 --dropout 0.0 --input_dropout 0.0 --weight_decay 2.747484129693843e-05 --d_model 448 --item_embedding_dim 448 --n_layer 2 --n_head 8 --label_smoothing 0.5 --stochastic_shared_embeddings_replacement_prob 0.0 --item_id_embeddings_init_std 0.09 --other_embeddings_init_std 0.015 --mlm_probability 0.1 --embedding_dim_from_cardinality_multiplier 3.0 --eval_on_test_set --seed 100 --use_side_information_features
```

Comment:
Hard to see the diff, but the only change is `--data_loader_engine nvtabular ` -> `--data_loader_engine merlin `.

Reply:
@sararb  do we still maintain `tf4rec_paper_experiments` ?  Meaning if we change the dataloader_engine to `merlin` is that gonna break anything?

Reply:
We are still maintaining `tf4rec_paper_experiments` because the `main` script is used in the integration tests ([here](https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/ci/test_integration.sh)). Changing the dataloader_engine won't break anything because the `merlin` and `nvtabular` aliases are both referring to the same`MerlinDataLoader` class.